tests/main/test_integration.py

```diff
@@ -123,22 +124,26 @@ def test_send_xem_scripts_command__gets_processed_in_fully_running_app(
     ]
     monitor_error_queue = test_process_monitor.get_fatal_error_reporter()
 
+    # Tanner (12/29/20): init with no bit file since Cloud9 does not have any real bit files and we are using a simulator
```

Comment:
minor note: "repository" or "local dev environment" may be better than saying Cloud9 specifically.  Cloud9 is just a specific IDE that we happen to like using at the moment--we might change to another IDE in the future